Sources/FluentUI_macOS/Theme/Tokens/ShadowInfo+AppKit.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+#if canImport(FluentUI_common)
+import FluentUI_common
+#endif
+import SwiftUI
+import AppKit
+
+public extension ShadowInfo {
```

Comment:
I'll add my obligatory "can we do this without extensions"? It's an extension of something in the same module which is less risky, but still adds some indirection. In this case, what if it was something like `ShadowFactoryAppKit` which creates NSShadows based on passed-in `ShadowInfo` objects?

Reply:
While that's very valid for Objective-C, and doubly-so when crossing modules, we heavily utilize same-module anonymous extensions in FluentUI's Swift code for organizing our code. This aligns well with Apple's own practices for SwiftUI, which extensively relies on extensions (pun slightly intended). This also reduces our public footprint and binary size by avoiding additional public factory classes.